### DISCUSSION TITLE
`.github_changelog_generator`  via parameter

### DISCUSSION THREAD

**Discussion by cilerler** [github-changelog-generator/github-changelog-generator#969] [Q&A] [answered]
Is it possible to set the `.github_changelog_generator` file as a parameter?  

I need to run the **github-changelog-generator** twice, one to update the changelog file, another one is for release notes _(which is simply the latest tag w/o changelog header, etc)_  

Is there a way to set it per command?

**Comment by olleolleolle** [comment]
That is not settable. Hope this helps!

**Reply by olleolleolle** [reply]
Haha, I was wrong, and the help can be seen in the CLI --help:

            --config-file CONFIG-FILE    Path to configuration file. Default is .github_changelog_generator.

**Comment by cilerler** [comment, marked as answer]
I found it, the following undocumented syntax handles it

```
github_changelog_generator --config-file .github_releasenotes_generator
```

**Reply by olleolleolle** [reply]
Ah, would you be up for including that in the CLI help? Or rather, where you first expected to find it.

Or, how was it not documented, you mean? Where did you learn about it and where did you first expect it to be?

**Reply by cilerler** [reply]
It should be on [this page](https://github.com/github-changelog-generator/github-changelog-generator/wiki/Advanced-change-log-generation-examples) so I would be able to identify it without asking.

I saw the list below in the CI's output and caught my attention.

```text
:date_format=>"%Y-%m-%d"
:output=>"CHANGELOG.md"
:base=>"HISTORY.md"
:issues=>true
:add_issues_wo_labels=>true
:add_pr_wo_labels=>true
:pulls=>true
:filter_issues_by_milestone=>true
:issues_of_open_milestones=>true
:author=>true
:unreleased=>true
:unreleased_label=>"Unreleased"
:compare_link=>true
:exclude_labels=>["duplicate", "question", "invalid", "wontfix", "Duplicate", "Question", "Invalid", "Wontfix", "Meta: Exclude From Changelog"]
:summary_labels=>["Release summary", "release-summary", "Summary", "summary"]
:breaking_labels=>["backwards-incompatible", "Backwards incompatible", "breaking"]
:enhancement_labels=>["enhancement", "Enhancement", "Type: Enhancement"]
:bug_labels=>["bug", "Bug", "Type: Bug"]
:deprecated_labels=>["deprecated", "Deprecated", "Type: Deprecated"]
:removed_labels=>["removed", "Removed", "Type: Removed"]
:security_labels=>["security", "Security", "Type: Security"]
:configure_sections=>{}
:add_sections=>{}
:issue_line_labels=>[]
:max_issues=>nil
:simple_list=>false
:ssl_ca_file=>nil
:verbose=>true
:header=>"# Changelog"
:merge_prefix=>"**Merged pull requests:**"
:issue_prefix=>"**Closed issues:**"
:summary_prefix=>""
:breaking_prefix=>"**Breaking changes:**"
:enhancement_prefix=>"**Implemented enhancements:**"
:bug_prefix=>"**Fixed bugs:**"
:deprecated_prefix=>"**Deprecated:**"
:removed_prefix=>"**Removed:**"
:security_prefix=>"**Security fixes:**"
:http_cache=>true
:require=>[]
:config_file=>".github_changelog_generator"
:user=>"cilerler"
:project=>"orange"
:token=>"hidden value"
```

**Reply by olleolleolle** [reply]
I did now add it to that page, which always risks becoming out of date.

    github_changelog_generator --help

will output stuff like:

        --config-file CONFIG-FILE    Path to configuration file. Default is .github_changelog_generator.
